adcm_aio_client/config/_refresh.py

```diff
@@ -30,7 +30,7 @@ def apply_local_changes(local: LocalConfigs, remote: ConfigData, schema: ConfigS
 
 def apply_remote_changes(local: LocalConfigs, remote: ConfigData, schema: ConfigSchema) -> ConfigData:
     if local.initial.id == remote.id:
-        return local.changed
+        return remote
```

Comment:
why? there aren't any conflicts, because initial data is the same

Reply:
with `local.changed` returned, `two_sessions_case_8` fails at last assertion

Reply:
this if works when both configs has same id

it's not the case with `two_sessions_case_8` where config save is involved: id of remote will be changed

Reply:
fixed